Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
@afnpires thanks a lot this pr, but there are too many changes at one time.

@damienbod @FabianGosebrink 
With AoT compilation & tree shaking we have to import only the modules we use, and not all the Angular files. Indeed, at the moment, for production, we have these sizes:
app: 603 KB
vendor: 318 KB
**tot: 921 KB**

After the pr, we have:
app: 154 KB
vendor: 1,23 MB
polyfills: 101 KB
**tot: 1,485 MB**

because tree shaking is not performed:

![image](https://cloud.githubusercontent.com/assets/14012361/21610460/7199b0b2-d1c8-11e6-8c31-8548552ad634.png)

Also, this is the old method of building an app. The Angular guide, uses only one bundle for the app and the vendors, but you can split polyfills, if you want.
To further reduce the size, there are three methods:
- Lazy loading of modules (with 'loadChildren', using angular-router-loader plugin); you can find an example in my sample app: https://github.com/robisim74/angular2localization/blob/gh-pages/app/app.routes.ts
- Google Closure Compiler for tree shaking and minification (but it's not simple): http://www.syntaxsuccess.com/viewarticle/tree-shaking-in-javascript
- Compression (e.g. .zip)
For further information, I suggest this article: http://blog.mgechev.com/2016/06/26/tree-shaking-angular2-production-build-rollup-javascript/

_TSLint_ for me is ok: I also use _Codelyzer_ and _reccomended_ rules: https://github.com/robisim74/Angular2SPAWebAPI/blob/master/src/Angular2SPAWebAPI/tslint.json

Greetings